### DISCUSSION TITLE
Switching between song and video

### DISCUSSION THREAD

**Discussion by roos-robert** [pear-devs/pear-desktop#2197] [Q&A] [answered]
Hi and thank you for this great app!

Small question, might not even be a feature but if it is I'd love to know if I am blind and cannot find it haha.

On the official YT music site you can listen to a song and then you always have the option to switch over to the music video for the song. In this player, for example if I start playing a song like this:

<img width="870" alt="image" src="https://github.com/th-ch/youtube-music/assets/5381910/969a0a28-4fde-4882-8356-12095d7465bf">

I cannot for the life of me se an option for this.

Is it just that this does not exist and that's why I cannot see it, or do I need to get my eyes checked haha?

Thanks in advance!

**Comment by TecnikOfficial** [comment, marked as answer]
reduce the size of the player window video / song switch button will appear 
if not working could try changing this options to native 
<details>
  <summary>Click to view screenshot</summary>
  <img src="https://github.com/user-attachments/assets/b16c164c-6aa1-4dd6-84be-2219cba0a918" alt="Screenshot">
</details>

**Reply by roos-robert** [reply]
Thank you so much for the reply, it's working great!
